src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +74,23 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
```

Comment:
Should we not also push the first post type here?
It seems like it can be in the same thing, but maybe you covered this in the other PR?

Reply:
Yes, that should be in the PR for completed actions. This one handles the new notification type.